File: Modul-01/c/PRAK110-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>
#include <math.h>

int main() {
    {    
    float alas, tinggi;
    printf("Diketahui :\nAlas = ");
    scanf("%f cm", &alas);

    printf("Tinggi = ");
    scanf("%f cm", &tinggi);
}


    float alas = 5, tinggi = 12;
    float sisiA = tinggi;
    float sisiB = sqrt(pow(tinggi, 2) + pow(alas, 2));
    float sisiC = alas;
    float keliling = sisiA + sisiB + sisiC;
    float luas = 0.5 * alas * tinggi;

    printf("\nJawab :\nSisi A = %.0f cm\n", sisiA);
    printf("Sisi B = %.0f cm\n", sisiB);
    printf("Sisi C = %.0f cm\n", sisiC);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm\n", luas);

    return 0;
}

File: Modul-01/python/PRAK110-2310817210009-MuhammadIbnuSina.py
import math

# Input dari terminal
alas_str = input("Diketahui :\nAlas = ")
tinggi_str = input("Tinggi = ")

alas_str = ''.join(filter(str.isdigit, alas_str))
tinggi_str = ''.join(filter(str.isdigit, tinggi_str))

# Mengonversi input menjadi float
alas = float(alas_str)
tinggi = float(tinggi_str)

# Menghitung panjang sisi-sisi segitiga
sisi_a = tinggi
sisi_b = math.sqrt(alas**2 + tinggi**2)

sisi_c = alas

# Menghitung keliling segitiga
keliling = sisi_a + sisi_b + sisi_c

# Menghitung luas segitiga
luas = 0.5 * alas * tinggi

# Menampilkan hasil
print(f"\nJawab :\nSisi A = {sisi_a:.0f} cm")
print(f"Sisi B = {sisi_b:.0f} cm")
print(f"Sisi C = {sisi_c:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm")
